src/shared/sam/cli/samCliStartApi.ts

```diff
@@ -71,10 +71,9 @@ export async function buildSamCliStartApiArguments(args: SamCliStartApiArguments
         ...(getLogger().logLevelEnabled('debug') ? ['--debug'] : []),
         '--template',
         args.templatePath,
```

Comment:
just noticed `eventPath` is not handled in samCliStartApi.ts, wonder if it should be

Reply:
Maybe? The only thing that could make sense is to have a sequence of events to hit the local server with, but that sounds like quite a bit of extra logic. A single event would be the same as local invoke so not particularly helpful.